.circleci/config.yml

```diff
@@ -0,0 +1,31 @@
+version: 2.1
+
+jobs:
+  lint:
+    docker:
+      - image: circleci/ruby:2.5.5-stretch
+    steps:
+      - checkout
+      - restore_cache: &restore_cache
+          name: Restore Bundle Cache
+          keys:
+            - codecademy-style-gem-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
+            - codecademy-style-gem-{{ .Branch }}
+            - codecademy-style-gem-master
+            - codecademy-style-gem-
+      - run: &bundle_install
+          command: bundle install --path vendor/bundle
+      - save_cache: &save_cache
```

Comment:
Not hurting anything but I don't think these `&task_name` jawns are necessary here, that syntax creates aliases that you can use elsewhere in the file

if you look at the other configs we have you'll see that we use the `&restore_cache` syntax outside of the `jobs` section, and use references like `*restore_cache` inside of the jobs section.

Reply:
You are right. Copy-pasta. And I wrote the thing I copied from myself!